lib/elixir/lib/regex.ex

```diff
@@ -101,6 +101,10 @@ defmodule Regex do
     * `:ungreedy` (U) - inverts the "greediness" of the regexp
       (the previous `r` option is deprecated in favor of `U`)
 
+    * `:export` (e) (from Erlang OTP 28.1 and Elixir 1.20) - uses an exported pattern
```

Comment:
```suggestion
    * `:export` (E) (from Erlang OTP 28.1 and Elixir 1.20) - uses an exported pattern
```

`e` is already used by Perl/PHP and has another meaning, let's use `E` to avoid any confusion.